### ISSUE TITLE
Cloud Compatibility of OpenELB 

### ISSUE THREAD

**Issue by AaronLinOops** [openelb/openelb#270]
Does openelb compatible with most cloud providers, such as aws, gcp, azure ..

**Comment by renyunkang** [comment]
We haven't tested this.

**Comment by mrpre** [comment]
It's impossible for user to run openelb on cloud,  because most cloud has it own way like `VPC`  to manage it network. Both L2 like ARP/VRRP are disabled.

**Comment by FeynmanZhou** [comment]
> It's impossible for user to run openelb on cloud, because most cloud has it own way like `VPC` to manage it network. Both L2 like ARP/VRRP are disabled.

Yes or not. I tested OpenELB on QingCloud VMs. It works by using Layer2. But I think you would better use the Cloud LoadBalancer plugin provided by the cloud providers. OpenELB is designed for on-premises Kubernetes.
